packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -241,7 +241,7 @@ library ChainAssertions {
         if (_isProxy) {
```

Comment:
Need to update the checkSystemConfig function to confirm that the superchain config address is correct